tool/tctl/common/tctl.go

```diff
@@ -346,7 +345,7 @@ func loadConfigFromProfile(ccf *GlobalCLIFlags, cfg *service.Config) (*authclien
 	if err := c.LoadProfile(cfg.TeleportHome, proxyAddr); err != nil {
 		return nil, trace.Wrap(err)
 	}
-	keyStore, err := client.NewFSLocalKeyStore(c.KeysDir)
+	keyStore, err := client.NewFSLocalKeyStore(cfg.TeleportHome)
```

Comment:
I'm not convinced that this doesn't break something.

What I would do is leave this the way it was (using `c.KeysDir`) and update `c.LoadProfile` (if necessary) to make sure it sets `c.KeysDir` appropriately.

Reply:
I've moved the setting of keysdir to the loadProfile function